FF1Lib/Flags.cs

```diff
@@ -547,6 +547,101 @@ public class Flags : IIncentiveFlags, IMapEditFlags, IScaleFlags, IFloorShuffleF
 			+ ((IncentivizeTitansTrove ?? true) ? 1 : 0)
 			+ ((IncentivizeSkyPalace ?? true) ? 1 : 0);
 
+		#region ExtensiveHints
+
+		public bool ExtensiveHints_Enable { get; set; } = true;

```

Comment:
Don't enable flags by default, this can mess up with presets

Reply:
done, was for testing...